### DISCUSSION TITLE
Styling of mailto: links

### DISCUSSION THREAD

**Discussion by kukimik** [srid/emanote#348] [General]
This was noticed when writing tests for #189.

Apart from external and internal links, we also have `mailto:` links. Currently these are classified as internal links and styled accordingly (bold text), which is probably not the expected result. On the other hand, adding an external link icon to a `mailto:` link also doesn't look right. I'm not sure how to treat them best.

There are also `javascript:` links, but that's another story.

It's not a big deal, perhaps not worth thinking about too much. I'm just leaving a note.

**Comment by srid** [comment]
Those shouldn't be classified as internal links but as normal links. Maybe we want to distinguish between 3 types of links: internal, external, normal.
